chain-abci/benches/tx.rs

```diff
@@ -86,7 +86,10 @@ fn init_chain_for(addresses: &Vec<RedeemAddress>) -> ChainNodeApp {
     );
 
     let params = InitNetworkParameters {
-        initial_fee_policy: LinearFee::new(Milli::new(1, 1), Milli::new(1, 1)),
+        initial_fee_policy: LinearFee::new(
+            Milli::try_new(1, 1).unwrap(),
```

Comment:
expect?

Reply:
it is the beche for testing, and it will never fail I think, so it doesn't matter to use unwrap.